[2024-06-12T03:43:42.488+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task2 manual__2024-06-12T03:43:41.648434+00:00 [queued]>
[2024-06-12T03:43:42.492+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task2 manual__2024-06-12T03:43:41.648434+00:00 [queued]>
[2024-06-12T03:43:42.492+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-06-12T03:43:42.493+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-06-12T03:43:42.493+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-06-12T03:43:42.499+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): task2> on 2024-06-12 03:43:41.648434+00:00
[2024-06-12T03:43:42.503+0000] {standard_task_runner.py:55} INFO - Started process 1004 to run task
[2024-06-12T03:43:42.508+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dag_with_callback', 'task2', 'manual__2024-06-12T03:43:41.648434+00:00', '--job-id', '305', '--raw', '--subdir', 'DAGS_FOLDER/example_dag_with_callback.py', '--cfg-path', '/tmp/tmpu2c4z2xq']
[2024-06-12T03:43:42.508+0000] {standard_task_runner.py:83} INFO - Job 305: Subtask task2
[2024-06-12T03:43:42.549+0000] {task_command.py:376} INFO - Running <TaskInstance: example_dag_with_callback.task2 manual__2024-06-12T03:43:41.648434+00:00 [running]> on host 3eba8accb13f
[2024-06-12T03:43:42.593+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_dag_with_callback
AIRFLOW_CTX_TASK_ID=task2
AIRFLOW_CTX_EXECUTION_DATE=2024-06-12T03:43:41.648434+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-12T03:43:41.648434+00:00
[2024-06-12T03:43:42.594+0000] {logging_mixin.py:120} INFO - Task 2 is completed
[2024-06-12T03:43:43.597+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-06-12T03:43:43.622+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=example_dag_with_callback, task_id=task2, execution_date=20240612T034341, start_date=20240612T034342, end_date=20240612T034343
[2024-06-12T03:43:43.635+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-06-12T03:43:43.654+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
